utils/scripts/daos-sudo-setup.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -uex
+
+# This script is used mainly by dockerfiles to setup a daos_server.
+# If the sudo package is installed that user will be given sudo access
+# for testing.
```

Comment:
I would argue that the script is mis-named given what the code is doing here.

Reply:
I will look at creating a better name.